examples/routing_service/routing_service_with_security/signFiles.bat

```diff
@@ -0,0 +1,5 @@
+set OPENSSL_CONF=..\..\..\resources\certAuthority\dsa01\ca\dsa01RootCa.cnf
+mkdir signed
+call openssl smime -sign -in governance.xml -text -out .\signed\governance.p7s -signer ..\..\..\resources\certAuthority\dsa01\ca\dsa01RootCaCert.pem -inkey ..\..\..\resources\certAuthority\dsa01\ca\private\dsa01RootCaKey.pem
```

Comment:
It's nice having this script here! I think in the future we can use CMake for this.

Reply:
Thanks. I have included those for generating `ecdsa`, `dsa` and `rsa`.